get_limit_datacard.sh

```diff
@@ -20,14 +21,17 @@ get_last_cat() {
 }
 
 pushd Datacard
-  if [ $dy_bkg_model = 0 ]; then 
-    python makeDatacardGGTT_new.py -o Datacard_${procTemplate}_${m}.txt --n-in-sideband $indir/../CatOptim/N_in_sidebands.json --MH $mh --MX $mx --MY $my --procTemplate ${procTemplate} --prune --do-res-bkg --sig-syst $indir/../Interpolation/systematics.json
+  if [ $dy_bkg_model = 0 -a $res_bkg_sys = 0 ]; then 
+    python makeDatacardGGTT_new.py -o Datacard_${procTemplate}_${m}.txt --n-in-sideband $indir/../CatOptim/N_in_sidebands.json --MH $mh --MX $mx --MY $my --procTemplate ${procTemplate} --prune --do-res-bkg --sig-syst ${sig_model}/systematics.json 
+
+  elif [ $dy_bkg_model = 0 -a $res_bkg_sys = 1 ]; then
+    python makeDatacardGGTT_new.py -o Datacard_${procTemplate}_${m}.txt --n-in-sideband $indir/../CatOptim/N_in_sidebands.json --MH $mh --MX $mx --MY $my --procTemplate ${procTemplate} --prune --do-res-bkg --sig-syst ${sig_model}/systematics.json --res-bkg-syst ${res_bkg_model}/systematics.json 
+
+  elif [ $dy_bkg_model = 1 -a $res_bkg_sys = 0 ]; then
+    python makeDatacardGGTT_new.py -o Datacard_${procTemplate}_${m}.txt --n-in-sideband $indir/../CatOptim/N_in_sidebands.json --MH $mh --MX $mx --MY $my --procTemplate ${procTemplate} --prune --do-res-bkg --sig-syst ${sig_model}/systematics.json --doABCD 
 
-    #python makeDatacardGGTT_new.py -o Datacard_${procTemplate}_${m}.txt --MH $mh --MX $mx --MY $my --prune --sig-syst ${sig_model}/systematics.json --res-bkg-syst ${res_bkg_model}/systematics.json --do-res-bkg
-    #python makeDatacardGGTT_new.py -o Datacard_${procTemplate}_${m}.txt --MH $mh --MX $mx --MY $my --prune --sig-syst ${sig_model}/systematics.json --res-bkg-syst ${res_bkg_model}/systematics.json --doABCD
   else
-    python makeDatacardGGTT_new.py -o Datacard_${procTemplate}_${m}.txt --n-in-sideband $indir/../CatOptim/N_in_sidebands.json --MH $mh --MX $mx --MY $my --procTemplate ${procTemplate} --prune --do-res-bkg --sig-syst $indir/../Interpolation/systematics.json --doABCD
-    #python makeDatacardGGTT_new.py -o Datacard_${procTemplate}_${m}.txt --n-in-sideband $indir/../CatOptim/N_in_sidebands.json --MH $mh --MX $mx --MY $my --procTemplate ${procTemplate} --prune --sig-syst ${sig_model}/systematics.json --res-bkg-syst ${res_bkg_model}/systematics.json --do-res-bkg --doABCD
-    #python makeDatacardGGTT_new.py -o Datacard_${procTemplate}_${m}.txt --MH $mh --MX $mx --MY $my --prune --sig-syst ${sig_model}/systematics.json --res-bkg-syst ${res_bkg_model}/systematics.json
```

Comment:
Why delete the commented out lines? Are they now included in the if clauses that you wrote above?

Reply:
I think here I have more if conditions to check do we want to do the signal systematics, or resonant bkg systematics, and the old ones are covered by the 4 conditions